test/data_static.jl

```diff
@@ -72,3 +72,83 @@ test_toyModel() = CoreModel(
     ["r1", "m1t", "m3t", "EX_m1(e)", "EX_m3(e)", "EX_biomass(e)", "biomass1"],
     ["m1[c]", "m3[c]", "m2[c]", "m1[e]", "m3[e]", "biomass[c]"],
 )
+
+function get_thermo_data()
```

Comment:
this doesn't need to be a function, just make it a global `Dict`.

Reply:
(the models are in functions because they're getting modified, this hopefully wouldn't :D )

Reply:
made it a constant dict